misc/release.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -e
+
+if [ -z "$1" ]; then
+    echo "Please provide the release version"
+fi
+
+VERSION="$1"
+TAG="v${VERSION}"
+
+#Ensure the release is correct
+git grep "version = \"${VERSION}\"" || (echo "version mismatch, check Cargo.toml" && false)
+
+echo "Craring version tag"
```

Comment:
wondering why the codespell is so bad at caching such simple typos ....
```suggestion
echo "Creating version tag"
```